### PR TITLE
Add Bank Transactions to GP cloud migration

### PR DESCRIPTION
**Pull request by ToddE123**
Re-adds the bank transactions to the GP cloud migration using general journal entries.

### CODE REVIEW

File: Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookMSTR.table.al [hunk 1]
```diff
@@ -178,94 +178,4 @@ table 40099 "GP Checkbook MSTR"
             Clustered = true;
         }
     }
-
-    procedure MoveStagingData()
```
Comment (nikolakukrika) [inline Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookMSTR.table.al:182]: We cannot move public methods - it is a breaking change. I love the refactoring done, but public method should stay and there should be a one line reference to the new method.

**Review by nikolakukrika — APPROVED** [review]

File: Apps/W1/HybridGP/app/src/codeunits/GPCloudMigration.codeunit.al [hunk 2]
```diff
@@ -134,4 +135,25 @@ codeunit 4025 "GP Cloud Migration"
         DataMigrationEntity.InsertRecord(Database::Item, HelperFunctions.GetNumberOfItems());
         exit(true);
     end;
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"Hybrid Cloud Management", 'OnInsertDefaultTableMappings', '', false, false)]
+    local procedure OnInsertDefaultTableMappings(DeleteExisting: Boolean; ProductID: Text[250])
```
[the remark this answers is not in the document]
Comment (ToddE123) [reply Apps/W1/HybridGP/app/src/codeunits/GPCloudMigration.codeunit.al:140]: none of the PRs submitted with this code has been accepted yet so it doesn't exist in MS main branch. Once one of the PRs are accepted, this code will merge with that.

File: Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookMSTR.table.al (hunk 1 above)
Comment (ToddE123) [reply Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookMSTR.table.al:182]: would the one line reference allow this method to still perform the migration or is it just a comment noting where the code was moved to?

**Review by JesperSchulz — APPROVED** [review]
Reviewed and approved by Nikola.

**Comment by ToddE123** [comment]
this is now up to date with main.

**Review by nikolakukrika — APPROVED** [review]

**Pull request merged**